wolfssl/internal.h

```diff
@@ -2409,6 +2409,19 @@ struct CRL_Monitor {
     #undef HAVE_CRL_MONITOR
 #endif
 
+/* PEM and DER possible */
+#define WOLFSSL_CRL_MONITORS_LEN (2)
```

Comment:
Should `WOLFSSL_CRL_MONITORS_LEN` be overridable? Seems like the existing CRL monitor code only support .der and .pem?

Reply:
I didn't like having the magic number `2` all around the code. There is no benefit of having this variable be overridable by the user so I didn't guard it with a `#ifndef`.